src/js/actions/layers.js

```diff
@@ -888,14 +890,14 @@ define(function (require, exports) {
         }
 
         var documentRef = documentLib.referenceBy.id(document.id),
-            layerRefs = layers
+            layerRefs = targetLayers
```

Comment:
the descriptor no longer needs all descendant layer group IDs in order to expand/collapse descendant groups recursively.